### DISCUSSION TITLE
Zigbee + Dirigera + Sensore umidita' terreno tuya viene riconosciuto come pulsante

### DISCUSSION THREAD

**Discussion by gianarb** [indomus/forum#3146] [GENERALE - Discussioni generiche]
Ciao!

Ho probabilmente deciso di avvicinarmi al protocollo zigbee nel modo piu' sbagliato visto che sono qui che ci perde tempo da settimane! Mi serve il vostro aiuto.

Ho iniziato con un gateway sonoff USB collegato al PC su cui gira home assistant, configurato zigbee2mqtt, smanaccato con con po' di codice per tradurre cosa questo sensore comunica

[NO LINK]

Ho poi notato che il gateway usb non era abbastanza potente per la destinazione finale del sensore e sono quindi passato a Ikea Dirigera con HomeKit device in Home Assistant ma quando aggiungo il sensore al gateway Ikea lo riconosciuto come pulsante e non come sensore quindi non riesco a vederne i valori di temperatura e umidita'.

Qualche idea sul come fare da qui in poi?
Grazie

**Comment by Scialla** [comment]
Sai che non ho capito? Di che sensore parli? Dirigera non è un coordinator universale e funziona solo con device ikea. Fossi in te tornerei alla sonoff (quale sonoff??) e, se hai problemi di distanza, aggiungerei un device alimentato che fa da router in mezzo

**Comment by gianarb** [comment]
Ciao!

Scusa per la spiegazione confusa ma rappresenta bene la mia attuale situazione.

La persona che mi ha consigliato Dirigera mi ha detto che non era limitato a device Ikea per quanto quelli sviluppati da loro siano supportati. Credo che sfortunatamente il mio device non sia supportato.

[NO LINK]

Questo è il mio sonoff a questo punto devo trovare una buona antenna e rendere il Dirigera.

Grazie

**Reply by Scialla** [reply]
Alla sonoff E non devi cambiare antenna, va bene così com'é, devi aggiungere un router, anche quello Ikea (hai presente quello con porta USB che si attacca al muro?) e metterlo in mezzo tra coordinator (antenna sonoff) e il tuo end device zigbee

**Comment by gianarb** [comment]
Grazie mille! Alla fine ho una soluzione che mi sembra funzionare, non sono ancora molto convinto quindi sfrutto questo post per fare un altra domanda.

Alla fine ho reso Dirigera perche' voglio un hub funzionale alla mia indole da smanettone e non troppo chiuso, sono riuscito a piazzare meglio il sonoff E e riesco a coprire bene il piano terra.

Volevo un aiuto per capire quali device posso fare da router. Ho 4 prese smart comprate su aliexpress ed ho visto che funzionano come router, ma il loro range di azione sembra molto basso, dal piano terra in cui sono installate non riesco a collegare il modo reliable un aqara sensore di temperatura che vorrei mettere al primo piano proprio sopra una delle prese.

Stavo quindi pensando di prendere uno o due relay da inserire in una scatola dove ho una presa che vorrei domotizzare solo che ho le seguenti domande, se l'aqara non riesce a collegarsi alla presa smart, riuscirebbe il relay da dentro la scatola? E l'eventuale relay farebbe da router? Perche' mi servirebbe per amplificare il segnale al primo piano. (nel caso quale relay? :) )

Grazie

**Reply by indomus** [reply]
usa un'altra zbdongle-e. meglio. https://indomus.it/guide/usare-sonoff-zbdongle-e-come-ripetitore-di-segnale-mesh-zigbee-router/

**Reply by gianarb** [reply]
Grazie mille!
